api/client.go

```diff
@@ -1466,3 +1577,14 @@ func (w *replicationStateStore) states() []string {
 	copy(c, w.store)
 	return c
 }
+
+// tokenSanityCheck will check for non-printable characters to prevent a call that will fail at the api
```

Comment:
We prefer to avoid the term "sanity check", see https://github.com/hashicorp/engineering-docs/blob/main/writing/style-guide.md#ableism

Reply:
Great call! Updating now and thanks for the doc!